### DISCUSSION TITLE
[HELP WANTED] International Portuguese variant is really needed?

### DISCUSSION THREAD

**Discussion by ferferga** [jellyfin/jellyfin-vue#2183] [Questions] [answered]
I believe we could have a single Portuguese translation with international alfabetization that works both for braziling and iberic mainlad speakers (just like we have with Spanish), but I'm not that knowledgeable of major differences between them.

This way we focus our efforts in having a single (but feature complete) portuguese language set.

If I have no feedback, I'll remove the Brazilian variant and just let the normal Portuguese live in Q2 2024.

*(A warning was added in Weblate to point Portuguese translations to this discussion)*

**Comment by tiagogaspar8** [comment]
Just to say my peace, I'd rather we split both languages. I'm Portuguese, and pt-pt compared to pt-br is quite different in some aspects.

Since translations are done in weblate, I can help maintain the pt-pt interface, as for the pt-br I can't speak for the pt-br.

I was wondering what is the the international version of Portuguese since I never heard of it. Can you point me to somewhere you read about it? maybe it is something, we learn stuff every day.

**Reply by ferferga** [reply, marked as answer]
I've been researching a bit more and it looks like it doesn't exist

I mentioned it that way because for Spanish, in Windows (language) and Word (corrector), there's an option that translated to English is something like "Spanish - Spain (traditional alphabetization)" alongside other options for Argentina, Mexico, etc... There's even a LATAM variant that covers all of them.

By the comment of @BRGustavoRibeiro, and all the resources I've been looking at after I asked for feedback (where there's a clear distinction between both, unlike Spanish), it looks like you guys don't have simple ways to "unformize" both languages.

Thank you both for your feedback, all I wanted was "on-terrain" feedback from native speakers (which will be always much better than any research I could do).

I'll leave this open for a bit so more people has the chance to chime in, but I think it's safe at this point to assume that both will be kept separated.

**Comment by BRGustavoRibeiro** [comment]
I really disagree with this decision - let's keep it separated. PT and PT-BR have really significant differences between each other.

There are many words that might create an "edit war" between portuguese and brazilian translators. For example:

[PT] Ecrâ
[BR] Monitor/Tela
[EN] Screen

[PT] Telemóvel
[BR] Celular 
[EN] Phone

[PT] Apelido
[BR] Sobrenome
[EN] Surname

These are just a few examples, but there are many other differences between both of them.

**Comment by ferferga** [comment]
@BRGustavoRibeiro @tiagogaspar8 Also, whenever you guys have a chance, both languages are not really well translated, so any help is appreciated! 🙌

**Comment by tiagogaspar8** [comment]
I'll get to it as soon as I can 😄

**Comment by ferferga** [comment]
Given native speakers assured me this is necessary and no feedback in favor of the removal was given, I'm closing this and **brazilian portuguese is kept**

**Comment by tiagogaspar8** [comment]
Thank you @ferferga,
And sorry I haven't yet been able to do translations!
